gladis/src/lib.rs

```diff
@@ -30,25 +30,51 @@
 //!     pub label: gtk::Label,
 //! }
 //!
-//! fn main() {
-//!     gtk::init().unwrap();
-//!     let _ui = Window::from_string(GLADE_SRC).unwrap();
-//! }
+//! gtk::init().unwrap();
+//! let _ui = Window::from_string(GLADE_SRC).unwrap();
 //! ```
 
-use gtk;
```

Comment:
Why is this removed? gtk is needed for the Gladis trait definition

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#single_component_path_imports

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#single_component_path_imports

Reply:
My bad, thanks!